bin/clever.js

```diff
@@ -421,19 +421,19 @@ function run () {
   // CONFIG COMMAND
   const config = lazyRequirePromiseModule('../src/commands/config.js');
   const configGetCommand = cliparse.command('get', {
-    description: 'Get the current configuration',
+    description: 'Display the current configuration',
     args: [args.configurationName],
   }, config('get'));
   const configSetCommand = cliparse.command('set', {
-    description: 'Set the configuration',
+    description: 'Edit one configuration setting',
     args: [args.configurationName, args.configurationValue],
   }, config('set'));
   const configUpdateCommand = cliparse.command('update', {
-    description: 'Update the configuration',
+    description: 'Edit multiple configuration settings at the same time',
```

Comment:
"at once" sounds better to me than "at the same time" in that context

Reply:
@divarvel Can you modify your PR with "at once"?
I'm about to to a release for a fix on `clever restart`, I'd love to merge this at the same time.

Reply:
I mosty use "at once" to mean "immediately", but it works as well here :+1:

Reply:
Since this was a pure "web UI" PR, I've created two commits, a squash-merge may be in order.

Reply:
I was about to "Y U NO SQUASH" :laughing: